client/mapView/MapView.cpp

```diff
@@ -115,9 +124,49 @@ void MapView::onMapScrolled(const Point & distance)
 
 void MapView::onMapSwiped(const Point & viewPosition)
 {
+	if(settings["adventure"]["smoothDragging"].Bool())
+		swipeHistory.push_back(std::pair<uint32_t, Point>(GH.input().getTicks(), viewPosition));
+
 	controller->setViewCenter(model->getMapViewCenter() + viewPosition, model->getLevel());
 }
 
+void MapView::postSwipe(uint32_t msPassed) {
+	if(!actions->dragActive)
+	{
+		if(swipeHistory.size() > 1)
+		{
+			Point diff = Point(0, 0);
+			std::pair<uint32_t, Point> firstAccepted;
+			uint32_t now = GH.input().getTicks();
+			for (auto & x : swipeHistory) {
+				if(now - x.first < 150) { // only the last 150 ms are catched

```

Comment:
Can you replace '150' with constant with descriptive name, e.g. `catchIntervalMilliseconds`?
Variable names are better documentation than comments